tests/NLog.UnitTests/LayoutRenderers/ExceptionTests.cs

```diff
@@ -429,17 +429,17 @@ public void InnerExceptionTest_Serialize()
             // Obsolete method requires testing until completely removed.
             logger.ErrorException("msg", ex);
             var lastMessage1 = GetDebugLastMessage("debug1");
-            Assert.StartsWith("{\"Message\":\"Wrapper2\"", lastMessage1);
-            Assert.Contains("\"InnerException\":{\"Message\":\"Wrapper1\"", lastMessage1);
+            Assert.StartsWith("{\"Type\":\"System.InvalidOperationException\", \"Message\":\"Wrapper2\"", lastMessage1);
```

Comment:
Why are those changes needed in the tests?

Reply:
> Adds "Type" as implicit property for Exception-objects